src/controllers/game.controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Controller,
+  Post,
+  Body,
+  BadRequestException,
+  Logger,
+} from '@nestjs/common';
+import { ApiTags, ApiResponse } from '@nestjs/swagger';
+import { GamesService } from 'src/Hangman/Application/Services/games.service';
+
+@Controller('games')
+@ApiTags('Games')
+export class GamesController {
+  private readonly logger = new Logger(GamesController.name);
+
+  constructor(private readonly gameService: GamesService) {}
+
+  @ApiResponse({ status: 201, description: 'Game created' })
+  @Post()
+  async startNewGame(
+    @Body()
+    {
+      gameId,
+      playerId,
+      wordToGuess,
+      maxGuesses,
+    }: {
+      gameId: string;
+      playerId: string;
+      wordToGuess: string;
+      maxGuesses: string;
+    },
+  ) {
+    try {
+      return await this.gameService.startNewGame(
+        gameId,
+        playerId,
+        wordToGuess,
+        parseInt(maxGuesses),
+      );
+    } catch (err) {
+      this.logger.error('startNewGame', err);
```

Comment:
found better practice is to instantiate Logger in class so it gets the right context, and is easier to mock?